src/crystal_lib/parser.cr

```diff
@@ -23,14 +28,17 @@ class CrystalLib::Parser
       @idx,
       [Clang::UnsavedFile.new("input.c", source)],
       flags,
-      Clang::TranslationUnit.default_options
+      Clang::TranslationUnit.default_options \
```

Comment:
You don't need a backslash when you simply move the pipe to the first line.

Reply:
TBH I find more readable to keep the operator at the beginning of the line but it's my personal opinion :) You'd prefer for the backslash to be removed here?

Reply:
Yeah, I'd prefer it that way, but it's a question of code style. It's fine to leave it your way.

Reply:
No problem! I'll change it that way 👍 You're probably reading & writing way more Crystal than I do so I prefer go the way you find to be the best on that kind of matter :-)